src/test/java/tutorspet/testutil/TypicalTutorsPet.java

```diff
@@ -28,7 +28,7 @@ public static TutorsPet getNoLinkTutorsPet() {
         for (Student student : TypicalStudent.getTypicalStudents()) {
             tp.addStudent(student);
         }
-        for (ModuleClass moduleClass : TypicalModuleClass.getTypicalModuleClasses()) {
+        for (ModuleClass moduleClass : TypicalModuleClass.getOnlyModuleClasses()) {
             tp.addModuleClass(new ModuleClassBuilder(moduleClass).withStudentUuids().build());
```

Comment:
I think this line can be simplified?
```suggestion
            tp.addModuleClass(moduleClass);
```